fluent.syntax/fluent/syntax/ast.py

```diff
@@ -14,25 +14,59 @@ class Visitor(object):
     descends into the children of the given AST node.
     '''
     def visit(self, value):
```

Comment:
I think I liked the split between `visit` and `visit_node`, but it's perfectly fine to combine them into a single method. I guess this is done for symmetry with `Transformer.visit`? Also, nit: rename `value` to `node`?

Reply:
Right, and also to follow the pattern of `ast` more closely. What really bothered me was that `Transformer` subclasses `Visitor` (for reasons I find odd but didn't debate), and thus had `visit_node`, but couldn't use it, really.